Testing/VelaptorTests/OpenGL/Buffers/RectGPUBufferTests.cs

```diff
@@ -63,21 +65,64 @@ public RectGPUBufferTests()
 
         this.mockGLService = new Mock<IOpenGLService>();
 
+        this.mockBatchSizeUnsubscriber = new Mock<IDisposable>();
+
         this.mockReactable = new Mock<IReactable>();
         this.mockReactable.Setup(m => m.Subscribe(It.IsAny<IReactor>()))
-            .Callback<IReactor>(reactor =>
+            .Returns<IReactor>(reactor =>
             {
                 reactor.Should().NotBeNull("it is required for unit testing.");
 
+                if (reactor.EventId == NotificationIds.GLInitializedId)
+                {
+                    // RETURN NULL TO SIMPLY IGNORE THIS EVENT ID
```

Comment:
Refactor to remove simply
```suggestion
                    // RETURN NULL TO IGNORE THIS EVENT ID
```